src/test/unit/regions/TMRegionTest.cpp

```diff
@@ -84,7 +84,7 @@ TEST(TMRegionTest, testSpecAndParameters) {
   Network net;
 
   // Turn on runtime Debug logging.
- //if (verbose)  LogItem::setLogLevel(LogLevel::LogLevel_Verbose);
+ //if (verbose)  NTA_LOG_LEVEL = LogLevel::LogLevel_Verbose;
```

Comment:
nit: It would be better if this were a function or at least a macro so that if we need to change the logic we don't need to change it everywhere it is used.

Note that with the current logic, setting the log level only affects the current cpp file and is not global.  That fact needs to be described someplace and there is no function to associate it with.

If the intent was to have the log level be set globally the variable NTA_LOG_LEVEL would have to be changed from static to extern and the actual variable would need to be defined in a .cpp someplace ... in Network.cpp perhaps.

In the NetworkAPI there are times that we would like to have this set globally so that debug messages on subordinate cpp's would be activated.  Currently each Region displays something on each iteration if this is activated globally.  Makes a nice trace.